src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1289,7 +1289,8 @@ internal enum ErrorCode
         // available 8041-8049
         ERR_InitializerOnNonAutoProperty = 8050,
         ERR_AutoPropertyMustHaveGetAccessor = 8051,
-        ERR_AutoPropertyInitializerInInterface = 8052,
+        ERR_AbstractEventHasAccessors = 8052,
+        // available 8053
```

Comment:
Please place all new C# 8 diagnostics in the region `diagnostics introduced for C# 8.0`

Reply:
> Please place all new C# 8 diagnostics in the region diagnostics introduced for C# 8.0

I am not aware of a requirement to do that.

---
In reply to: [285274626](https://github.com/dotnet/roslyn/pull/35756#discussion_r285274626) [](ancestors = 285274626)

Reply:
However, I was under impression that ERR_AutoPropertyInitializerInInterface was added as part of the DIM work, but that is not the case. Since we cannot reuse error numbers for errors that were shipped, I will not reuse this number for ERR_AbstractEventHasAccessors.

---
In reply to: [285274982](https://github.com/dotnet/roslyn/pull/35756#discussion_r285274982) [](ancestors = 285274982,285274626)

Reply:
Placing a C# 8 diagnostic in a region labeled `diagnostics introduced in C# 4 and earlier` is misleading, which is why I asked you to move it.

---
In reply to: [285274982](https://github.com/dotnet/roslyn/pull/35756#discussion_r285274982) [](ancestors = 285274982,285274626)